Description: this crate is a verbatim copy of gix-packetline for technical
reasons, it always has the blocking-io feature pre-selected which conflicts
with async-io, so it's not possible to enable the latter.

Index: gix-packetline-blocking/Cargo.toml
===================================================================
--- gix-packetline-blocking.orig/Cargo.toml
+++ gix-packetline-blocking/Cargo.toml
@@ -62,7 +62,6 @@ default-features = false
 version = "1.0.34"
 
 [features]
-async-io = []
 blocking-io = []
 default = ["blocking-io"]
 serde = [
